rotkehlchen/balances/manual.py

```diff
@@ -45,30 +46,44 @@ def get_manually_tracked_balances(
         db: 'DBHandler',
         balance_type: BalanceType | None = None,
         include_entries_with_missing_assets: bool = False,
+        use_main_currency: bool = False,
 ) -> list[ManuallyTrackedBalanceWithValue]:
```

Comment:
why does this need the use_main_currency? Can't you detected if main_currency != A_USD?